### DISCUSSION TITLE
October 1 "Global day of action for Iran"

### DISCUSSION THREAD

**Discussion by Setareh4IRAN** [WeAreMahsaAmini/Core#224] [Ideas]
Hey, I was thinking how can we have the best possible contents and highest impact from this day.
Maybe we can get ready from before , If we know someone who can record voices, good videos, anyone with drones ..
or maybe contacting reporters , planning a special act worldwide as a symbol of " Iranians Union" .. What do you think?! Let's discuss.

**Comment by ghost** [comment]
Hey I'm in Toronto and there will be a few media outlets present in our protest. I know of a few people who are also planning on recording videos and photos to share on social media. It would be cool if we all could share our footage from all over the world the next day. Do you have anything specific in mind?

**Reply by Setareh4IRAN** [reply]
Not anything specific , I just wanted to discuss about it together
